pkg/cmd/roachtest/tests/failover.go

```diff
@@ -140,9 +229,7 @@ func runFailoverNonSystem(
 	t.Status("creating workload database")
 	_, err = conn.ExecContext(ctx, `CREATE DATABASE kv`)
 	require.NoError(t, err)
-	_, err = conn.ExecContext(ctx, `ALTER DATABASE kv CONFIGURE ZONE USING `+
-		`num_replicas = 3, constraints = '[-node1, -node2, -node3]'`)
-	require.NoError(t, err)
+	constrainConfig(t, ctx, conn, `DATABASE kv`, 3, []int{1, 2, 3}, 0)
```

Comment:
nit: Not sure I think this is an improvement. It's harder to read and understand, and less flexible, just to avoid copying a few lines of code.